index.html

```diff
@@ -4,16 +4,69 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Shafiu | Portfolio</title>
+    <link rel="stylesheet" href="./index.css">
 </head>
 <body>
     <div>
-        <header>
-
-        </header>
-        <section>
-            
-        </section>
+        <nav class="navbar">
+            <div class="nav__logo">
+                <a href="#">SHAFIU</a> 
+            </div>
+            <div class="nav__hamburger">
+                <img src="./images/hamburger.png" alt="not internet">
+            </div>
+        </nav>
```

Comment:
- [ ] Kindly I recommend you to use `header` tag for toolbar section, Typically the `header `tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.